changelog.d/3-bug-fixes/stern-api-check

```diff
@@ -0,0 +1 @@
+Remove overly restricte api check
```

Comment:
arguably it's overly restrictive to make sure we're running into this early, and not when we *have* to make the version bump.  but that's clearly arguable, yes.  :)